src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -77,8 +77,12 @@ class Initializer(config: Config) extends SparkSessionWrapper {
     if (config.consumerDatabaseName != config.databaseName) {
       logger.log(Level.INFO, "Initializing Consumer Database")
       if (!spark.catalog.databaseExists(config.consumerDatabaseName)) {
-        val createConsumerDBSTMT = s"create database if not exists ${config.consumerDatabaseName} " +
-          s"location '${config.consumerDatabaseLocation}'"
+        val createConsumerDBSTMT = if (!config.isLocalTesting) {
```

Comment:
can we not set the location as a relative path in the config for local testing and not need a different logic branch for this?